emission/analysis/classification/inference/mode.py

```diff
@@ -191,47 +271,53 @@ def getGroundTruthMode(self, section):
 # 20. both start and end close to train station
 # 21. both start and end close to airport
   def updateFeatureMatrixRowWithSection(self, featureMatrix, i, section):
-    featureMatrix[i, 0] = section['distance']
-    featureMatrix[i, 1] = (section['section_end_datetime'] - section['section_start_datetime']).total_seconds()
+    if 'distance' in section.keys():
+      featureMatrix[i,0] = section['distance']
+    else:
+      featureMatrix[i, 0] = easf.calDistance(section['start_loc']['coordinates'], section['end_loc']['coordinates'])  #####would need to calculate this
```

Comment:
None of the sections have a distance, besides the outlier that we add in (which is why I say in the code comments I need to make a distance outlier using the new model instead of `distance`). In the new model, the only sort of thing related to it that we have is a start location and an end location, so the only way to get any sort of distance is just to get the linear distance between them.